engine/classes/Elgg/Database/EntityTable.php

```diff
@@ -140,13 +141,16 @@ function rowToElggStar($row) {
 	/**
 	 * Loads and returns an entity object from a guid.
 	 *
-	 * @param int    $guid The GUID of the entity
-	 * @param string $type The type of the entity. If given, even an existing entity with the given GUID
-	 *                     will not be returned unless its type matches.
+	 * @param int    $guid             Entity GUID
+	 * @param string $type             Entity type. If given, even an existing entity with the given GUID
+	 *                                 will not be returned unless its type matches.
+	 * @param string $subtype          Entity subtype (use "" for default subtype). If given, even an existing
+	 *                                 entity with the given GUID will not be returned unless this matches.
+	 * @param bool   $ignore_condition If true, hidden/disabled entities may be found
 	 *
 	 * @return \ElggEntity The correct Elgg or custom object based upon entity type and subtype
 	 */
-	function get($guid, $type = '') {
+	function get($guid, $type = '', $subtype = null, $ignore_condition = false) {
```

Comment:
@hypeJunction can you elaborate on the use case for this? I added it based on [your suggestion](https://github.com/Elgg/Elgg/pull/8942#discussion_r39630192), but I can't think of a good one.